### PR TITLE
basic multi_client - 2 peer pgw test

### PR DESCRIPTION
**Pull request by vijay-hirani**
I have made  a basic multi client - 2 pgw target test. Before I put some more 'meat' no the test, I want make check my approach is correct. Appreciate any comments on how to alter/improve it .

### CODE REVIEW

File: test/ergw_test_lib.erl [hunk 1]
```diff
@@ -23,7 +23,8 @@
 	 gtp_context_inc_restart_counter/1,
 	 gtp_context_new_teids/1, gtp_context_new_teids/3,
 	 make_error_indication_report/1]).
--export([start_gtpc_server/1, stop_gtpc_server/1, stop_gtpc_server/0,
+-export([start_gtpc_server/1, start_gtpc_server/2,
+	 stop_gtpc_server/1, stop_gtpc_server/0, stop_gtpc_server_node/1,
```
Comment (ebengt) [inline test/ergw_test_lib.erl:27]: For symmetry reason I would suggest the name
stop_gtpc_server/1,

File: test/pgw_proxy_SUITE.erl [hunk 2]
```diff
@@ -18,6 +18,8 @@
 -include("ergw_pgw_test_lib.hrl").
 
 -define(TIMEOUT, 2000).
+-define(NO_OF_CLIENTS, 6). %% No of IP clients for multi session tests
```
Comment (ebengt) [inline test/pgw_proxy_SUITE.erl:21]: When I see NO I think of none, or no such or negations. If number is too long, how about num ?

**Review by ebengt — APPROVED** [review]
Looks good to me.

File: test/pgw_proxy_SUITE.erl (hunk 2 above)
Comment (vijay-hirani) [reply test/pgw_proxy_SUITE.erl:21]: changed it to NUM_OF_CLIENTS.

File: test/ergw_test_lib.erl (hunk 1 above)
Comment (vijay-hirani) [reply test/ergw_test_lib.erl:27]: Done

**Comment by RoadRunnr** [comment]
looks mostly ok, but IPv6 support is missing

**Comment by RoadRunnr** [comment]
@vijay-hirani was this included in #202 ?

**Comment by vijay-hirani** [comment]
Yes, it is. please close this branch. Thanks

**Comment by RoadRunnr** [comment]
included in #202

**Pull request closed without merging**